csrc/quantization/gptq_marlin/gptq_marlin.cu

```diff
@@ -390,6 +391,17 @@ __device__ inline void sub_zp(typename ScalarType<scalar_t>::FragB& frag_b,
   frag_b[1] = __hsub2(frag_b[1], zp);
 }
 
+template <typename scalar_t>
+__device__ inline void sub_zpf(typename ScalarType<scalar_t>::FragB& frag_b,
```

Comment:
nit: could this be templated to remove duplication with `sub_zp`? 

not critical